components/experimental/src/measure/category/mod.rs

```diff
@@ -0,0 +1,37 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use core::marker::PhantomData;
+
+use crate::measure::measureunit::MeasureUnit;
+
+pub mod area;
+pub mod length;
+pub mod mass;
+pub mod volume;
+
+pub trait MeasureUnitCategory {}
```

Comment:
@sffc ,  I do not know why do we need these right now?

Reply:
https://github.com/unicode-org/icu4x/pull/6535/commits/3ad67fc7e6043bc62d9c576b4dd45c519a0559fc

Reply:
not *strictly* required right now, but if you want to add any functionality to `CategorizedMeasureUnit` you'll need this